src/adapters/criteo.js

```diff
@@ -134,6 +123,50 @@ var CriteoAdapter = function CriteoAdapter() {
     return bidObject;
   }
 
+  function _buildBidObject(bidResponse, slot) {
+    let bidObject;
+    if (bidResponse) {
+
+      // map the common fields
+      bidObject = bidfactory.createBid(1);
+      bidObject.bidderCode = _bidderCode;
+      bidObject.cpm = bidResponse.cpm;
+
+      // in case of native
+      if (slot.nativeCallback && bidResponse.native) {
+
+        if (typeof slot.nativeCallback !== "function") {
+          utils.logError("Criteo bid: nativeCallback parameter is not a function");
+        } else {
+
+          // store the callbacks in a global object
+          window.criteo_pubtag.native_slots = window.criteo_pubtag.native_slots || {};
+          window.criteo_pubtag.native_slots[""+bidObject.adId] = { callback: slot.nativeCallback, nativeResponse: bidResponse.native };
+
+          // this code is executed in an iframe, we need to get a reference to the
+          // publishertag in the main window to retrieve native responses and callbacks.
+          // it doesn't work with safeframes
+          bidObject.ad =  "<script type=\"text/javascript\">";
```

Comment:
please use string literals for constructing the templates. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals